core/src/main/scala/cats/syntax/list.scala

```diff
@@ -27,6 +27,6 @@ final class ListOps[A](val la: List[A]) extends AnyVal {
     * }}}
     */
   def toNel: Option[NonEmptyList[A]] = NonEmptyList.fromList(la)
-  def groupByNel[B](f: A => B): Map[B, NonEmptyList[A]] =
+  def groupByNel[B : Order](f: A => B): Map[B, NonEmptyList[A]] =
```

Comment:
shouldn't the type here also be `SortedMap`?

Reply:
Yes, it should. Working on it.

Reply:
Well... current status: yak-shaving inside `NonEmptyTraverse` doctests, where `SortedMap` needs an `Apply` instance.

Reply:
I've pushed something that addresses this. I've fixed the `NonEmptyTraverse` doctests by upcasting to a vanilla `Map` for which there's an `Apply` instance. I'm not completely satisfied with this approach, but it's the best I've got right now.

Reply:
I think making an Apply and Traversable for SortedMap is probably out of scope, so I think that's okay.